### PR TITLE
Removed validation for characters

### PR DESCRIPTION
**Pull request by AkashAryal**
### Summary 

Temporary hotfix to remove input text checking on the form

### CODE REVIEW

File: client/src/ui/CUreviewsGoogleLogin.tsx [hunk 1]
```diff
@@ -64,8 +64,11 @@ export default class CUreviewsGoogleLogin extends Component<Props, { lastVerific
 
   getRedirectURI = () => {
     if (window.location.host.includes("localhost")) {
+      console.log("http://" + window.location.host + "/auth/")
```
Comment (AkashAryal) [inline client/src/ui/CUreviewsGoogleLogin.tsx:67]: Oops. But its okay this will be clean up in another PR. This is just a hotfix

**Review by juliejeong — APPROVED** [review]
Approve!
